src/main/java/uk/gov/hmcts/reform/sendletter/data/LetterMapperFactory.java

```diff
@@ -14,18 +15,19 @@
 final class LetterMapperFactory {
 
     public static final RowMapper<LetterStatus> LETTER_STATUS_MAPPER = new LetterStatusMapper();
+    public static final RowMapper<NotPrintedLetter> NOT_PRINTED_LETTER_MAPPER = new NotPrintedMapper();
 
-    private static final class LetterStatusMapper implements RowMapper<LetterStatus> {
+    static ZonedDateTime getDateTime(ResultSet rs, String columnLabel) throws SQLException {
+        Timestamp timestamp = rs.getTimestamp(columnLabel);
 
-        private ZonedDateTime getDateTime(ResultSet rs, String columnLabel) throws SQLException {
-            Timestamp timestamp = rs.getTimestamp(columnLabel);
+        if (timestamp == null) {
+            return null;
+        }
 
-            if (timestamp == null) {
-                return null;
-            }
+        return timestamp.toInstant().atZone(ZoneOffset.UTC);
+    }
 
-            return timestamp.toInstant().atZone(ZoneOffset.UTC);
-        }
+    private static final class LetterStatusMapper implements RowMapper<LetterStatus> {
```

Comment:
this whole block from line 18 looks weird. Basically it's just moving method outside of the inner class